### DISCUSSION TITLE
Export only a part of the wiki?

### DISCUSSION THREAD

**Discussion by yfrindel** [rfennell/AzurePipelines#924] [Q&A] [answered]
Hi

Thanks for this tool. But Is it possible to export only a part of the wiki (a specific folder with all of their subfolders)?
Our Wiki has a lots of pages, and I would like to export only a specific folder (e.g : https://mycompany.com/Dev/_git/DevelopmentOps.wiki?path=%2FBuild-process, where "Build-process" is the name of the only folder that I want to export)

**Comment by rfennell** [comment, marked as answer]
Given the way I have wrapper'ed the [underlying AzureDevOps.WikiPDFExport tool](https://github.com/MaxMelcher/AzureDevOps.WikiPDFExport) that is used to export the WIKI as PDF it takes either the path of root of the WIKI repo or to a single File.

You could test the AzureDevOps.WikiPDFExport tool locally and see if you specify a path to a subfolder in your WIKI repo and see if that delivers what you want. If it does we could look at altering the task to allow something similar. let me know how it goes

**Reply by yfrindel** [reply]
Thx for the reply. In my opinion, a new field with the path of the folder to export could be very useful. So, if this new field is empty, all the wiki is exported, and if a folder path is specified, only this folder is exported

**Reply by rfennell** [reply]
Need to check it works as expected first, but yes if it does a sensible update

**Reply by yfrindel** [reply]
I tried what you suggested, and it works 👍 

![image](https://user-images.githubusercontent.com/22955005/103909927-8fd8c800-5104-11eb-909b-13302a84af58.png)

As you can see in my screenshot, I have specified the subfolder of the wiki repo, and only this subfolder is used for the PDF export !
So could you implement that please?

**Reply by rfennell** [reply]
We then, I will convert this discussion to be an issue and look to make the change

**Reply by yfrindel** [reply]
Thank you Richard ! 👍

**Reply by rfennell** [reply]
Shipped as 1.4.16

**Reply by yfrindel** [reply]
Brilliant ! Do you know when this new version will be available on the marketplace please? (it's still the 1.3.8 version currently)

**Reply by rfennell** [reply]
Sorry, it was late in the day. I thought I had approval the final release, I had not, just done it now should be live in a few minutes

**Reply by yfrindel** [reply]
no problem. Thank you very much for all. And have a nice day ;)
